### PR TITLE
#8 add write method overload with input stream

### PR DESCRIPTION
**Pull request by fgrutsch**

### CODE REVIEW

**Comment by fgrutsch** [comment]
Refs #8

File: src/main/scala/com/globo/bigdata/ghdfs/HdfsManager.scala [hunk 1]
```diff
@@ -34,6 +35,19 @@ class HdfsManager(hdfs: FileSystem) {
 
   def write(hadoopPath: Path): FSDataOutputStream = create(hadoopPath: Path)
 
+  def write(hadoopPath: Path, is: InputStream): Boolean = {
```
Comment (regismachado) [inline src/main/scala/com/globo/bigdata/ghdfs/HdfsManager.scala]: About the InputStream argument, can you change the variable name from "is" to another thing like "input"?

File: src/main/scala/com/globo/bigdata/ghdfs/HdfsManager.scala [hunk 2]
```diff
@@ -34,6 +35,19 @@ class HdfsManager(hdfs: FileSystem) {
...
+    val fsOutputStream = create(hadoopPath)
```
Comment (regismachado) [inline src/main/scala/com/globo/bigdata/ghdfs/HdfsManager.scala]: Change the call of the create(..)  to the write(..).

Today create and write are similar, but maybe in the future, we will add new behaviours to the write

**Review by regismachado — CHANGES_REQUESTED** [review]

**Comment by dmvieira** [comment]
Please solve conflicts @fg-devs

**Comment by fgrutsch** [comment]
@dmvieira resolved

**Comment by dmvieira** [comment]
Thx!!!

**Pull request merged**